extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -108,7 +108,7 @@ private static function getCachedSvg( $name ) {
 	 *
 	 * @return string
 	 */
-	public static function renderText( $fields, $recursionDepth = 0 ) {
+	public static function renderText( $fields, $inContentLang = false, $recursionDepth = 0 ) {
```

Comment:
also this method is 50 lines long, do you think we could refactor it into smaller pieces?